linkedin_scraper/person.py

```diff
@@ -159,44 +184,71 @@ def scrape_not_logged_in(self, close_on_complete=True, retry_limit=10):
             retry_times = retry_times + 1
 
         # get name
-        self.name = driver.find_element_by_id("name").text.strip()
+        self.name = driver.find_element_by_xpath(
+            "/html/body/main/section[1]/section/section[1]/div/div[1]/div[1]/h1").text.strip()
```

Comment:
I try really hard not to use full xpaths like this, because it's less responsive to change. Is there another way to get this element?

Reply:
`self.name = driver.find_element_by_class_name("top-card-layout__title")`

this will work instead of full xpath. Please let me know if it looks good.

Reply:
we also can segregate and make steps like

```
topcard = driver.find_element_by_class_name("top-card-layout")
self.name = topcard.find_element_by_class_name("top-card-layout__title")
```
please let me know which one you prefers?